CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Thanks to: @btoconnor, @bugsounet, @JasonStieber, @khassel, @kleinmantara and @W
 - [calendar] Added config option "showEndsOnlyWithDuration" for default calendar
 - [compliments] Added `specialDayUnique` config option, defaults to `false` (#3465)
 - [weather] Provider weathergov: Use `precipitationLast3Hours` if `precipitationLastHour` is `null` (#3124)
+- [core] Added `DOM_OBJECTS_UPDATED` notification each time the DOM is re-rendered via `updateDom` (#3534)
```

Comment:
This is not in the section for the new release.

@rejas Can we fix this without a new PR?

Reply:
I can fix it directly on `develop` or we put it on the imaginary todo-list for next release ...